Login/register css

calmoon808 opened the pull request:
Seperated registration survey in to parts, added CSS, changed how password was getting passed to avoid a google security error I was getting.

calmoon808 committed:
pre mpre merge

calmoon808 committed:
merged

calmoon808 committed:
Changed how password gets passed to be more secure, seperated registration survey into seperate views, added CSS

danieltichiyama commented on src/components/LoginComponent/LoginComponent.jsx:
```diff
@@ -48,11 +48,15 @@ class LoginComponent extends Component {
 
   render() {
+    if (this.props.loginError === true){
+      alert("Email and password combination to not match.");
```
"to" to "do"

jkhirano replied:
Alert initially works when my password is wrong. But as I try to delete so I can re-enter it, the alert keeps popping up with every delete key pressed. The very last letter doesn't delete so I can't re-enter password.

danieltichiyama commented on src/components/RegisterComponent/RegisterComponent.jsx:
```diff
@@ -51,7 +51,12 @@ class RegisterComponent extends Component {
+    return this.setState({ password: this.props.password}, () => {
+      delete this.state.confirm_password;
+      this.props.takingSurvey(this.state);
+    })
```
does this have to go into a .then() statement???

danieltichiyama commented on src/index.js:
```diff
@@ -22,7 +22,7 @@ const enhancer = composeEnhancers(applyMiddleware(ReduxThunk)); //applyMiddlewar
 
 ReactDOM.render(
-  <Provider store={store}>
+  <Provider store={store}z>
```
delete z

danieltichiyama replied:
please delete z

danieltichiyama commented on src/reducers/index.js:
```diff
@@ -175,6 +185,9 @@ let reducer = (store = initialStore, action) => {
       return Object.assign({}, store, { images: action.payload }); 
       
+    case BCRYPT:
+      return Object.assign('', store, { bcrypt: action.payload })
```
change from '' to {}

jkhirano commented on src/components/RegisterSurveyComponent/RegisterSurveyComponent.jsx:
```diff
@@ -46,243 +82,395 @@ class RegisterSurveyComponent extends Component {
+                  this.setState({ activity_level_id: e.target.value });
+                }}
+              />
+              <label htmlFor="veryActive">veryActive</label>
```
Change to: <label htmlFor="veryActive">Very Active</label>

jkhirano commented on src/components/RegisterSurveyComponent/RegisterSurveyComponent.jsx:
```diff
@@ -46,243 +82,395 @@ class RegisterSurveyComponent extends Component {
+        {this.state.goalSelect ? (
+          <>
+            <h2>Goal</h2>
+            <li>
```
The list/button is getting cut off by the navbar in some views (galaxy / iphone). I think you need to put the goal list elements in a div and then apply flex to fix it so it's responsive.

jkhirano replied:
Flex box should be applied to all other list divs as well

jkhirano reviewed (commented):
css looks good :) only have a few things to change..I'm not sure if fixing the password popup is necessary since we're short on time. If it's not fixed, we'll just have to make sure that doesn't show up during the presentation.

calmoon808 committed:
register survey component removed and combined with register component fixed other issues

calmoon808 committed:
Merge branch 'dev' into login/register_css

calmoon808 commented:
K everything rerouted now. changed the register bcrypt process back to its original state. register survey component is now part of register component. fixed the delete bug after failed login attempt. Hopefully nothing else comes up

danieltichiyama commented on src/actions/index.js:
```diff
@@ -130,17 +131,19 @@ export const actionsLoadActivity = () => async dispatch => {
+      dispatch({
+        type: LOGIN_ERROR
+      });
+      return false
```
should it return false here or should it return the dispatch?  also, maybe we ought to console.log the error like the other actions?

danieltichiyama commented on src/components/LoginComponent/LoginComponent.jsx:
```diff
@@ -33,9 +32,15 @@ class LoginComponent extends Component {
+          alert("Email and password combination to not match.");
+          return false;
+        }
+      })
```
should this be a .then??

danieltichiyama commented on src/components/RegisterComponent/RegisterComponent.module.scss:
```diff
@@ -1,33 +1,47 @@
+$turquoise: #04c9b5;
+$red: #fc4a1a;
```
oh, shit, nice.

calmoon808 committed:
Merge branch 'dev' into login/register_css
